src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+html {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.header__1200 {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  height: 60px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo__container {
+  display: flex;
+  margin-left: 50px;
+}
+
+.nav {
+  display: flex;
+  margin-right: 50px;
+
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  height: 60px;
+
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item:not(:last-child) {
+  display: flex;
+  margin-right: 20px;
+}
+
+.nav__link {
+  padding: 23px 0;
+  position: relative;
+  text-decoration: none;
+  text-transform: uppercase;
+  text-align: center;
+  color: black;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active:hover {
+  color: #00acdc;
+}
```

Comment:
It's redundant here cause the `is-active` link must have a blue color. So delete this.